app/templates/core/create-account-modal/setup-account-panel.jade

```diff
@@ -24,6 +24,11 @@
           span {{ $t("signup.teacher_list_resource_hub_5") }}
       span(v-html="$t('signup.teacher_additional_questions', {supportEmail})")
 
+    br
+    div.text-center
+      input#subscribe-input(type="checkbox" name="subscribe" checked=true)
+      span.spl {{ $t("signup.teacher_email_announcements") }}
+
```

Comment:
Use a label to group these together like in the code removed from BasicInfoView:
```
-              label
 -                input#subscribe-input(type="checkbox" name="subscribe" checked=view.signupState.get('signupForm').subscribe[0] === 'on')
 -                span(data-i18n="signup.email_announcements")
```

This makes it so the entire text label is clickable instead of just the checkbox itself.